spec/features/user/dashboard_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.feature "View Dashboard", type: :feature do
+  let(:user) { FactoryBot.create :user }
+  let(:admin) { FactoryBot.create :admin }
+
+  scenario "User can access his dashboard from user dropdown menu" do
+    sign_in user
+    visit root_path
+
+    click_on(text: user.first_name, class: 'dropdown-toggle')
+    click_on I18n.t('navbar.drop-down.dashboard')
+
+    expect(page).to have_css('h1', text: I18n.t('dashboard.title'))
+  end
+
+  scenario "Admin can access any user's dashboard" do
+    sign_in admin
+    visit dashboard_user_path(user)
+
+    expect(page).to have_css('h1', text: I18n.t('dashboard.title'))
+  end
+
+end
```

Comment:
Maybe add a test to check if the dashboard contains the user's content. (Add the user's name to the Dashboard page and test for its presence). In order to check that the correct page is opened.

Reply:
Ok, this wasn't originally speced but I checked with Dima and it's ok if the headline contains the user's name. I'll make the changes.